lib/Memory/Memorable.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <EEPROMex.h>
+#include <EEPROMVar.h>
+
+namespace Memory {
```

Comment:
It looks very weird. Try:
```cpp
enum class EEPROM_Addreses{
  WIFI = 0x0,
  GAME = 0x69
};
```

Reply:
It's generate extras problems whenever i try add something to address. 
`namespace Memory {
  const int WIFI_START = 0x0;
}`

Reply:
I think that the `static_cast` is not so bad. If you really don't want to put this into `enum` let's check the name of this variable. Maybe `Memory::WIFI_ADDRESS` or `MemoryAddresses::WIFI`?